src/sagemaker/jumpstart/model.py

```diff
@@ -850,8 +859,8 @@ def register_deploy_wrapper(*args, **kwargs):
 
         return model_package
 
-    @lru_cache
-    def _get_benchmark_data(self, config_name: str) -> Dict[str, List[str]]:
+    @lru_cache(typed=True)
```

Comment:
This is unnecessary, this function will only accept string as input anyways. `@lru_cache(typed=True)` is used when input can be multiple types